### PR TITLE
Email sender

### PR DESCRIPTION
**Pull request by alanhala**
### Summary
- added credentials to secrets.yml and then ignored this file in the repo
- created the mailer model with the view
- route for the controller new method

### Trello card
https://trello.com/c/mY0hSAuz/14-crear-email-con-los-ultimos-10-articulos-creados

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,6 @@
 /log/*
 !/log/.keep
 /tmp
+
+# Ignore scret tokens
+/config/secrets.yml
```
Comment (mdesanti) [inline .gitignore]: Do not ignore this file

File: app/mailers/last_articles_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class LastArticlesMailer < ActionMailer::Base
+  default from: "alan.halatian@wolox.com.ar"
+
+  def last_articles(user)
+    @user =  user
+    @articles = Article.last(10)
+    mail subject: "Last Articles List",
```
Comment (mdesanti) [inline app/mailers/last_articles_mailer.rb]: I18n

File: app/controllers/articles_controller.rb [hunk 3]
```diff
@@ -31,6 +31,15 @@ def destroy
     redirect_to articles_path
   end
 
+  def last_articles
+    if user_signed_in?
+      LastArticlesMailer.last_articles(current_user).deliver
+      redirect_to articles_path, :flash => { :notice => I18n.t(:email_sent) }
```
Comment (mdesanti) [inline app/controllers/articles_controller.rb]: Use this sintax: `:lash: { notice: I18n.t(:email_sent) }`

Comment (alanhala) [inline app/controllers/articles_controller.rb]: @mdesanti I fixed that in the merged branch

**Comment by mdesanti** [comment]
:+1:

**Pull request merged**